frontend/src/index.tsx

```diff
@@ -37,6 +39,7 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
           },
         ]}
       />
+      <ToastContainer />
```

Comment:
Let's create a separate component for the toaster and use ToastContainer in that component